### DISCUSSION TITLE
No matches for kind "ImagePolicy" in version "image.toolkit.fluxcd.io/v1alpha2"

### DISCUSSION THREAD

**Discussion by thuyhoang-gp** [fluxcd/flux2#4352] [Q&A] [answered]
Hi all,

I have recently been troubled with the upgrade helm release due to an error as below.

```
Helm upgrade failed: [resource mapping not found for name: "<<release_name>>" namespace: "flux-system" from "": no matches for kind "ImagePolicy" in version "image.toolkit.fluxcd.io/v1alpha2" ensure CRDs are installed first, resource mapping not found for name: "<<release_name>>" namespace: "flux-system" from "": no matches for kind "ImageRepository" in version "image.toolkit.fluxcd.io/v1alpha2" ensure CRDs are installed first] Last Helm logs: preparing upgrade for resetting values to the chart's original version
```

It seems like image.toolkit.fluxcd.io/v1alpha2 is outdated. I am asking for help because I cannot find any documentation related to this issue.

Thank you.

**Comment by makkes** [comment, marked as answer]
These API versions have been [removed from image-reflector-controller a year ago](https://github.com/fluxcd/image-reflector-controller/blob/main/CHANGELOG.md#0230) and they have been [deprecated for more than a year prior to that removal](https://github.com/fluxcd/image-reflector-controller/blob/main/CHANGELOG.md#0110). Please upgrade your manifests to use the most recent version v1beta2.
